app/ui/src/main.ts

```diff
@@ -2,6 +2,7 @@ import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { environment } from './environments/environment';
 import { AppModule } from './app';
+import './rxjs.imports';
```

Comment:
Ideally, this should come BEFORE the `import { AppModule } ...` LOC, since that module might include at some point async code relying on RxJS operators (ideally not, but yeah... this is Syndesis and anything can happen here)

Reply:
Actually, I'd suggest to bump this LOC to the very top of the file.